examples/gno.land/r/demo/boards/z_4_filetest.gno

```diff
@@ -341,7 +341,7 @@ func main() {
 //                 "Base": {
 //                     "@type": "/gno.RefValue",
 //                     "Escaped": true,
-//                     "ObjectID": "336074805fc853987abe6f7fe3ad97a6a6f3077a:2"
+//                     "ObjectID": "purePkg:336074805fc853987abe6f7fe3ad97a6a6f3077a:2"
```

Comment:
for object from pure package, make objectID with a prefix "purePkg", so it's possible to distinct with object from realm. this does not change the current realm object persistence. but it looks ugly.

Reply:
@thehowl any idea for this?

Reply:
maybe `r:xxx` and `p"xxx` is good.